datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearch.tsx

```diff
@@ -119,7 +119,12 @@ export const EmbeddedListSearch = ({
         return refetch(variables);
     };
 
-    const { data, loading, error } = useGetSearchResults({
+    const {
+        data,
+        loading,
+        error,
+        refetch: realRefetch,
```

Comment:
might make more sense to rename the other one `refetchForDownload`

Reply:
i likey